plugins/newsql/newsql_evbuffer.c

```diff
@@ -520,37 +521,54 @@ static int newsql_read_evbuffer(struct sqlclntstate *clnt, void *b, int l, int n
     return have / l;
 }
 
-static int newsql_pack_hb(uint8_t *out, void *arg)
+static int newsql_pack_hb(struct sqlwriter *writer, void *arg)
```

Comment:
Need corresponding change in fastsql

Reply:
Thanks for pointing out. It's in https://bbgithub.dev.bloomberg.com/comdb2/bb-plugins/pull/262.